app-backup/btrbk/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>candrews@integralblue.com</email>
+        <name>Craig Andrews</name>
+        <description>Proxied maintainer; set to assignee in all bugs</description>
```

Comment:
&lt;description&gt; isn't used anymore. Per GLEP67, the first maintainer is the default bug assignee.